app/src/main/java/com/zeapo/pwdstore/crypto/PasswordCreationActivity.kt

```diff
@@ -235,6 +236,11 @@ class PasswordCreationActivity : BasePgpActivity(), OpenPgpServiceConnection.OnB
                                 snackbar(message = getString(R.string.password_creation_duplicate_error))
                                 return@executeApiAsync
                             }
+
+                            if (!URI(path).normalize().path.contains(repoPath)) {
+                                snackbar(message = getString(R.string.message_error_destination_outside_repo))
+                                return@executeApiAsync
+                            }
```

Comment:
why not use this from the previous PR? Declare it as an extension function and re-use it.

```kotlin
private fun isInsideRepository(file: File): Boolean {
    return file.canonicalPath.contains(getRepositoryDirectory(this).canonicalPath)
}
```

Reply:
I don't really know how to create an extension function... jeje... But I'll search it and change it.

Reply:
Look at the `Extensions.kt` file in the repo, it's pretty straightforward.

Reply:
I think I did it :), if you can please check.